### ISSUE TITLE
Offsets-Tracker Broken

### ISSUE THREAD

**Issue by PerlMonker303** [open-telemetry/opentelemetry-go-instrumentation#250]
### Describe the bug

I tried running the "make offsets" command to create the offset_results.json file, however I get the following output:
```
$ make offsets
cd offsets-tracker; OFFSETS_OUTPUT_FILE="../pkg/inject/offset_results.json" go run main.go
go: Discovering available versions
go: Downloading version 1.21.0
go: Analyzing binary for version 1.21.0
2023/08/09 11:24:49 error while fetching offsets: decoding dwarf section info at offset 0x0: too short
exit status 1
make: *** [Makefile:86: offsets] Error 1
```

What I tried:
- building the docker image from /offsets-tracker directory and run it myself, but encountered the same issue.
- using the original package ([keyval-dev/offsets-tracker](https://github.com/keyval-dev/offsets-tracker/blob/master/README.md)) - same error;
- removing the stdLibOffsets and only keeping the grpcOffsets executed successfully.

Any ideas?

### Environment

 - OS: Debian GNU
 - Go Version: 1.20
 - Version: latest

### To Reproduce

1. Clone the repository
2. Run "make offsets"

### Expected behavior

The offset_results.json file is created. The execution of the "make offsets" command leads to no errors.

**Comment by MrAlias** [comment]
Reproduced failure in our build system: https://github.com/open-telemetry/opentelemetry-go-instrumentation/actions/runs/5847008758/job/15854626858?pr=253

**Comment by MrAlias** [comment]
I believe this is relevant: https://github.com/golang/go/issues/61714

**Comment by MrAlias** [comment]
Closed by #256
